db/schema.rb

```diff
@@ -75,6 +75,15 @@
 
   add_index "deliveries", ["supplier_id"], name: "index_deliveries_on_supplier_id", using: :btree
 
+  create_table "documents", force: true do |t|
+    t.string "name"
+    t.binary "content", limit: 16.megabyte
```

Comment:
I use the filename extension in this case.
IMHO storing the "unsafe" content-type provided by the browser is not much better than using the extension of the file.